test/App/DocsSpec.hs

```diff
@@ -0,0 +1,40 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+module App.DocsSpec (
+  spec,
+) where
+
+import App.Docs (newIssueUrl, userGuideUrl)
+import Control.Exception (throwIO)
+import Data.Maybe (fromJust)
+import Data.Text (Text)
+import Network.HTTP.Req (
+  GET (GET),
+  MonadHttp (handleHttpException),
+  NoReqBody (NoReqBody),
+  bsResponse,
+  req,
+  responseStatusCode,
+  useHttpsURI,
+ )
+import Test.Hspec (Expectation, Spec, describe, it, shouldBe)
+import Text.URI (mkURI)
+
+instance MonadHttp IO where
+  handleHttpException = throwIO
```

Comment:
I think you want [`runReq :: MonadIO m => HttpConfig -> Req a -> m a`](https://hackage.haskell.org/package/req-3.9.1/docs/Network-HTTP-Req.html#v:runReq) in the `Expectation` to avoid needing to define this orphan instance. Check out the examples under `req`'s documentation.